js/main.js

```diff
@@ -25,8 +25,12 @@
                 id = currentPhoto.id;
                 title = currentPhoto.title;
                 secret = currentPhoto.secret;
-                picUrl = "https://farm"+farm+".staticflickr.com/"+server+"/"+id+"_"+secret+"_z.jpg";
-                $('.image-slider-wrapper ul').append("<li class='active imgArray' ><a data-lightbox='image-set'" + " href='" + picUrl + "'><img class='active' "+ " src='" +picUrl+"'/></a></li>");
+                picUrl = "https://farm" + farm + ".staticflickr.com/" + server + "/" + id + "_" + secret + "_z.jpg";
+
+                if(i%4 === 0){
+                    $('.image-slider-wrapper').append("<div class='row'></div>");
+                }
+                $('.image-slider-wrapper>.row:last-child').append("<div class='col-xs-3'><a data-lightbox='image-set'" + " href='" + picUrl + "'><img class='img-responsive' "+ " src='" +picUrl+"'/></a></div>");
```

Comment:
Just as an example, can you have 3 different grids: 2 columns for mobile (`col-xs-6`), 4 columns for desktop (`col-md-3`), and 12 columns for wide-screens (`col-lg-1`)? I'd like for you to try working creating a single HTML structure, but have bootstrap change the display. 

Let me knew if you have any issues.